### PR TITLE
Handle missing doc when loading About page

### PR DESCRIPTION
**Pull request by garethbowen**
medic/cht-core#7100

# Code review checklist

- [x] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [x] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [x] Tested: Unit and/or e2e where appropriate
- [x] Internationalised: All user facing text
- [x] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: webapp/tests/karma/ts/modules/about/about.component.spec.ts [hunk 1]
```diff
@@ -120,6 +120,12 @@ describe('About Component', () => {
     expect(spySubscriptionsUnsubscribe.callCount).to.equal(1);
   });
 
+  it('handles missing partners resource - #7100', async(async () => {
```
Comment (latin-panda) [inline webapp/tests/karma/ts/modules/about/about.component.spec.ts:123]: Can you please add a test where it's catching and printing the 404 error? Thanks!

**Review by latin-panda — COMMENTED** [review]
Nice! Just one question in line

File: webapp/tests/karma/ts/modules/about/about.component.spec.ts (hunk 1 above)
Comment (garethbowen) [reply webapp/tests/karma/ts/modules/about/about.component.spec.ts:123]: Done!

**Review by latin-panda — APPROVED** [review]
Looks great! Thank you!✨

**Pull request merged**